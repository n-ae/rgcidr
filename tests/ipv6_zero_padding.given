2001:db8::1
2001:0db8::1
2001:00db8::1
2001:000db8::1
2001:0000db8::1
02001:db8::1
002001:db8::1
0002001:db8::1
00002001:db8::1
2001:0db8:0000:0000:0000:0000:0000:0001
2001:0db8:0:0:0:0:0:1
2001:db8:0000:0000:0000:0000:0000:0001
2001:db8::0001
2001:db8::001
2001:db8::01
2001:db8::1
::1
::0001
::001
::01
2001:db8:85a3:0000:0000:8a2e:0370:7334
2001:db8:85a3::8a2e:370:7334
2001:0db8:85a3:0000:0000:8a2e:0370:7334
0001:0002:0003:0004:0005:0006:0007:0008
1:2:3:4:5:6:7:8
