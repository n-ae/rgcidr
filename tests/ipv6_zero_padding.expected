2001:db8::1
2001:0db8::1
2001:00db8::1
2001:000db8::1
2001:0000db8::1
02001:db8::1
002001:db8::1
0002001:db8::1
00002001:db8::1
2001:0db8:0000:0000:0000:0000:0000:0001
2001:0db8:0:0:0:0:0:1
2001:db8:0000:0000:0000:0000:0000:0001
2001:db8::0001
2001:db8::001
2001:db8::01
2001:db8::1
2001:db8:85a3:0000:0000:8a2e:0370:7334
2001:db8:85a3::8a2e:370:7334
2001:0db8:85a3:0000:0000:8a2e:0370:7334
